.github/workflows/washboard.yml

```diff
@@ -0,0 +1,68 @@
+name: washboard
+
+on:
+  push:
+    paths: 
+      - 'washboard-ui/**'
```

Comment:
Is there a reason not to just use `washboard`?

Reply:
Surfacing the fact that it is a UI, much like `wash` vs `wash-cli`